drivers/atwinc15x0/atwinc15x0_netdev.c

```diff
@@ -482,11 +727,47 @@ static int _atwinc15x0_set(netdev_t *netdev, netopt_t opt, const void *val,
             } else {
                 return max_len;
             }
+        case NETOPT_SCAN:
+            if (!IS_USED(MODULE_ATWINC15X0_DYNAMIC_SCAN)) { goto forward_to_netdev; }
```

Comment:
Since all cases return directly, the `goto` statement can be replaces easily by a `break` and calling 
```c
return netdev_eth_set(netdev, opt, val, max_len);
```
after the the switch/case construct.